ironfist/src/cpp/combat/combat.cpp

```diff
@@ -390,6 +390,12 @@ void SpecialAttackGraphics(army *attacker, army *target)
 		v27 = 10;
 		v18 = 10;
 	}
+	if (attacker->creatureIdx == CREATURE_CYBER_BEHEMOTH) {
```

Comment:
I don't really know what it is right now actually, it's either the offset or size of the sprite I guess?
But yeah I guessed that this fixes the problem